README.md

```diff
@@ -100,6 +103,112 @@ overview of all settings configurable through the CLI.
 | `S3.HostBucketEnabled`               | Enables bucket rewriting in the router               | -                                 | `--s3.hostBucketEnabled`           | `RENTERD_S3_HOST_BUCKET_ENABLED`               | `s3.hostBucketEnabled`              |
 | `S3.KeypairsV4 (DEPRECATED)`                      | V4 keypairs for S3                                   | -                                 | -                                  | -            | `s3.keypairsV4`                     |
 
+### Single-Node Setup
+
+A single-node setup involves running all components (bus, worker, and autopilot)
+on the same machine. This is ideal for testing, development, or small-scale
+deployments. This setup is the default when running `renterd` without any flags.
+
+### Cluster Setup
+
+In a cluster setup, the bus, worker, and autopilot run on separate nodes. This
+setup is ideal for large-scale deployments where you want to horizontally scale
+your renter. The worker nodes can be spread across multiple machines, and the
+autopilot can be run on a separate machine.
+
+#### Bus Node Configuration
+
+The bus is the only node that exposes the UI. To run the bus separately, the
+autopilot and worker have to be disabled using the `--autopilot.enabled` and
+`--worker.enabled` flags. The only other requirement to run a bus is the (walet)
+seed.
+
+#### Worker Node Configuration
+
+To configure the worker as a standalone node, the autopilot has to be disabled
+using the `--autopilot.enabled` flag, and the bus has to be disabled. There's no
+flag to explicitly disable the `bus`, it's implied by configuring a remote
+address for the bus using the `--bus.remoteAddr` and `--bus.remotePassword`
+flags. When the bus is remote, the worker has to be configured with an external
+address of the form http://<worker-ip>:<port>, on localhost however this can be
```

Comment:
`http://<worker-ip>:<port>` doesn't render correctly without making it a code block